release/dashboard/mem_check.py

```diff
@@ -15,6 +15,8 @@
     client = JobSubmissionClient("http://127.0.0.1:8265")
     job_id = client.submit_job(
```

Comment:
I have edited this test in my anyscale jobs PR to allow it to have a configurable working dir. Feel free to copy https://github.com/ray-project/ray/blob/2aff537a924a6f5befeaf6f477867a2c3dc31854/release/dashboard/mem_check.py